docs/api-reference/3d-tiles/tile-3d-header.md

```diff
@@ -1,4 +1,4 @@
-# Tile3DHeader (Experimental)
+# Tile3DHeader
 
 > The 3D tile loaders are still under development. If you are interested in early access, please open an issue.
```

Comment:
While this is experimental and APIs may change, the "please open an issue" comment is no longer correct. The 3d-tiles module is available on npm now.